### PR TITLE
created mysql database with tables

### PR DESCRIPTION
**Pull request by EduardDumitrescul**

### CODE REVIEW

File: database/init.sql [hunk 1]
```diff
@@ -0,0 +1,62 @@
+use vibesyncdb;
+
+CREATE TABLE User (
+    id INT PRIMARY KEY,
+    nickname VARCHAR(255)
+);
+
+CREATE TABLE Event (
```
Comment (skpha13) [inline database/init.sql:8]: Would it be a good idea to also have a `postedDate` to let the user know when the event was created ?

File: database/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# MySQL Database Setup
+You can find username, root password, user password and database name inside (docker-compose.yml)
+
+## How to install
+- open terminal in this directory
+- create the container:
+```docker-compose up -d```
+
+## Useful commands
```
Comment (skpha13) [inline database/README.md:9]: It would be useful to also have commands on how to run the sql files

File: database/init.sql (hunk 1 above)
Comment (EduardDumitrescul) [reply database/init.sql:8]: done

File: database/README.md (hunk 2 above)
Comment (EduardDumitrescul) [reply database/README.md:9]: init.sql is now automatically run when creating the container

Comment (skpha13) [reply database/README.md:9]: Another thing that came to mind while we are at it :). .NET backend is code first when creating the database and populating it with data. So does this become redundant?

**Pull request merged**